network/f5/bigip/snmp/mode/poolstatus.pm

```diff
@@ -197,8 +196,9 @@ sub manage_selection {
         my $result = $options{snmp}->map_instance(mapping => $mapping->{$map}, results => $snmp_result->{$branch_name}, instance => $instance);
         $result->{Name} = '';
         foreach (split /\./, $instance) {
-            $result->{Name} .= chr  if ($_ >= 32 && $_ <= 126);
+            $result->{Name} .= chr if ($_ >= 32 && $_ <= 126);
         }
+        $result->{Name} =~ s/^.//;
```

Comment:
Looks really odd to fix it this way.   Do we have on the root cause of this first random char ? Is this chr provoking it

Reply:
Not that odd. The names are built from oid indexes that are ASCII translation of the names. But there always is a random charactere in first position.

Reply:
roger that, just wonder why this char is popping ... let's merge to let everybody get access to this fix

Reply:
Yea I know, hope its not a bug from my f5 box...

Reply:
Need to test it against our snmp simulator on friday. we got one walk from other user ;)

Reply:
Already tested, and there is no pools or virtualservers :/